integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java

```diff
@@ -179,35 +173,35 @@ public void prepare() throws Exception {
    */
   @AfterAll
   public static void staticUnPrepare() throws Exception {
-    if (FULLTEST) {
-      if (domain != null) {
-        domain.destroy();
-        TestUtils.deleteWeblogicDomainResources(domainNS1);
-        if (BaseTest.SHARED_CLUSTER) {
-
-          String image = System.getenv("REPO_REGISTRY")
-              + "/weblogick8s/"
-              + domainNS2
-              + "-image:" + currentDateTime;
-          String cmd = "docker rmi -f " + image;
-          TestUtils.execOrAbortProcess(cmd, true);
-        }
-        String cmd = "docker rmi -f " + domainNS2 + "-image:" + currentDateTime;
+    Assumptions.assumeTrue(FULLTEST);
+    if (domain != null) {
+      domain.destroy();
+      TestUtils.deleteWeblogicDomainResources(domainNS1);
+      if (BaseTest.SHARED_CLUSTER) {
+
+        String image = System.getenv("REPO_REGISTRY")
+            + "/weblogick8s/"
+            + domainNS2
+            + "-image:" + currentDateTime;
+        String cmd = "docker rmi -f " + image;
         TestUtils.execOrAbortProcess(cmd, true);
       }
-      if (operator != null) {
-        operator.destroy();
-      }
-      try {
-        uninstallWebHookPrometheusGrafanaMySql();
-      } catch (Exception ex) {
-        LoggerHelper.getLocal().log(Level.INFO,
-            "Exception caught while uninstalling webhook/prometheus/Grafana/Mysql " + ex.getMessage());
-      }
-
-      tearDown(new Object() {}.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString());
-      LoggerHelper.getLocal().log(Level.INFO,"SUCCESS");
+      String cmd = "docker rmi -f " + domainNS2 + "-image:" + currentDateTime;
+      TestUtils.execOrAbortProcess(cmd, true);
```

Comment:
If its a SHARED_CLUSTER both docker rmi -f commands get executed or line # 189 is supposed to be in a else block?

Reply:
yes, if shared cluster it also tags image to [ocipath]/imagename as well, so need to delete both images [ocipath]/imagename in else block for shared cluster and "imagename" in general